File: jimmer-ddd-java/src/main/java/org/morecup/jimmerddd/core/sqlclient/AggregationEntityProps.java
package org.morecup.jimmerddd.core.sqlclient;

import org.babyfish.jimmer.internal.GeneratedBy;
import org.babyfish.jimmer.meta.ImmutableType;
import org.babyfish.jimmer.meta.TypedProp;
import org.babyfish.jimmer.sql.ast.table.Props;
import org.babyfish.jimmer.sql.ast.table.PropsFor;

@GeneratedBy(
        type = AggregationEntity.class
)
@PropsFor(AggregationEntity.class)
public interface AggregationEntityProps extends Props {
    TypedProp.Scalar<AggregationEntity, Boolean> ID_PRE_LOADED = 
        TypedProp.scalar(ImmutableType.get(AggregationEntity.class).getProp("idPreLoaded"));
}

File: jimmer-ddd-kotlin/src/main/kotlin/org/morecup/jimmerddd/core/sqlclient/AggregationEntityProps.kt
@file:Suppress("warnings")
@file:GeneratedBy(type = AggregationEntity::class)

package org.morecup.jimmerddd.core.sqlclient

import kotlin.Boolean
import kotlin.Suppress
import org.babyfish.jimmer.`internal`.GeneratedBy
import org.babyfish.jimmer.kt.toImmutableProp
import org.babyfish.jimmer.meta.TypedProp

@GeneratedBy(type = AggregationEntity::class)
public object AggregationEntityProps {
    public val ID_PRE_LOADED: TypedProp.Scalar<AggregationEntity, Boolean> =
            TypedProp.scalar(AggregationEntity::idPreLoaded.toImmutableProp())
}
